libraries/ESP8266WiFi/src/ESP8266WiFiGeneric.h

```diff
@@ -121,6 +121,7 @@ class ESP8266WiFiGenericClass {
     public:
         int hostByName(const char* aHostname, IPAddress& aResult);
         int hostByName(const char* aHostname, IPAddress& aResult, uint32_t timeout_ms);
+        int hostByNameAsync(const char* aHostname, IPAddress& aResult, uint8_t* waiting = NULL);

```

Comment:
`waiting` should be a `bool` since it only has `0` or `1` states.

Reply:
Done in 37b35d78c2acc701b3755640674ed068c193188a